beamer/tests/agent/unit/test_handlers.py

```diff
@@ -27,6 +27,27 @@
 from beamer.typing import FillId, Termination
 
 
+@pytest.mark.parametrize("claimable", [True, False])
+def test_claim_after_expiry(claim_request_extension, claimable):
+    context, config = make_context()
+    valid_until = TIMESTAMP - claim_request_extension
```

Comment:
Nitpick: we should add
```
    assert valid_until > 0
```
here, similar to what we have in `test_ignore_expired`.